scenarios/kubernetes_e2e.py

```diff
@@ -298,6 +298,8 @@ def main(args):
         runner_args.append('--stage=%s' % args.stage)
 
 
+    cluster = args.cluster or 'e2e-gce-%s-%s' % (
+        os.environ['NODE_NAME'], os.getenv('EXECUTOR_NUMBER', 0))
```

Comment:
Right but that's a complex change that is hard to prediect -- will this cause problems with soak jobs that reuse a cluster?

I would augment the current behavior for now and then switch it up after code-freeze?